formatTest/typeCheckedTests/expected_output/attributes.re

```diff
@@ -60,45 +60,45 @@ let x = "hello" [@onHello];
 
 let x = "hello" [@onHello];
 
-let x = "hello" ^ ("goodbye" [@onGoodbye]);
+let x = "hello" ^ "goodbye" [@onGoodbye];
```

Comment:
Currently, are parsed as if they are final arguments to a function, and apply to that entire function call. The parens were unnecessary. I still don't like this convention (I'd rather have attributes come before the function), but we can fix that later.